applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -599,6 +599,78 @@ void BaseSolidElement::CalculateOnIntegrationPoints(
 
             rOutput[point_number] = StrainEnergy;  // 1/2 * sigma * epsilon
         }
+    } else if ( rVariable == ERROR_INTEGRATION_POINT ) {
+        const SizeType number_of_nodes = GetGeometry().size();
+        const SizeType dimension = GetGeometry().WorkingSpaceDimension();
+        const SizeType strain_size = mConstitutiveLawVector[0]->GetStrainSize();
+
+        KinematicVariables this_kinematic_variables(strain_size, dimension, number_of_nodes);
+        ConstitutiveVariables this_constitutive_variables(strain_size);
+
+        // Create constitutive law parameters:
+        ConstitutiveLaw::Parameters Values(GetGeometry(),GetProperties(),rCurrentProcessInfo);
+
+        // Set constitutive law flags:
+        Flags& ConstitutiveLawOptions=Values.GetOptions();
+        ConstitutiveLawOptions.Set(ConstitutiveLaw::USE_ELEMENT_PROVIDED_STRAIN, UseElementProvidedStrain());
+        ConstitutiveLawOptions.Set(ConstitutiveLaw::COMPUTE_STRESS, false);
+        ConstitutiveLawOptions.Set(ConstitutiveLaw::COMPUTE_CONSTITUTIVE_TENSOR, true);
+
+        // Reading integration points
+        const GeometryType::IntegrationPointsArrayType& integration_points = GetGeometry().IntegrationPoints(  );
+
+        //Calculate Cauchy Stresses from the FE solution
+        std::vector<Vector> sigma_FE_solution(number_of_nodes);
+        Variable<Vector> variable_stress = CAUCHY_STRESS_VECTOR;
```

Comment:
what is `CAUCHY_STRESS_VECTOR`?

Reply:
Is the vector containing (following Voigt notation) the Cauchy stresses